exercise-15/src/com/nayema/Actor.java

```diff
@@ -0,0 +1,15 @@
+package com.nayema;
+
+public class Actor {
+    private static String[] name = {"Arnold Schwarzenegger", "Will Smith", "Anglenia Jolie", "Brad Pitt"};
+
+    private Actor() {
```

Comment:
- The private constructor was not used by a factory method to create an Actor.
- The Actor private constructor did not enforce the requirement that all Actors must have a name set at the start when they are initialized.